src/gui/advancedsettings.cpp

```diff
@@ -609,6 +617,23 @@ void AdvancedSettings::loadAdvancedSettings()
     // Announce to all tiers
     m_checkBoxAnnounceAllTiers.setChecked(session->announceToAllTiers());
     addRow(ANNOUNCE_ALL_TIERS, tr("Always announce to all tiers"), &m_checkBoxAnnounceAllTiers);
+
+    // Peer Turnover
+    m_spinBoxPeerTurnover.setMinimum(0);
+    m_spinBoxPeerTurnover.setMaximum(100);
+    m_spinBoxPeerTurnover.setValue(session->peerTurnover());
+    m_spinBoxPeerTurnover.setSuffix(tr(" %"));
+    addRow(PEER_TURNOVER, tr("Percentage of peers to disconnect"), &m_spinBoxPeerTurnover);
```

Comment:
Maybe "Peer turnover percentage" would be a better name. It would also be nice to link that to https://www.libtorrent.org/reference-Settings.html#peer_turnover as the setting alone seems a little confusing.

Reply:
Yes - will do both.